tests/e2e/ia-production/ia-prod-base.js

```diff
@@ -6,7 +6,9 @@ const localURL = 'https://archive.org/details/';
 const books = [
   'birdbookillustra00reedrich', // publicDomain
   // 'pianoservicingtu00rebl_0', // borrowable,
-  // 'adventuresoftoms00twaiiala' // has chapter markers
+  // 'adventuresoftoms00twaiiala', // has chapter markers
+  // 'gendaitankashu00meijuoft', // Right to Left book
+  // 'gov.uspto.patents.application.10074026', // multiple files
```

Comment:
should we open these up soon?

doesn't have to be a part of this pr.

Reply:
With help from @jbuckner and Hank comments, we should have most if not all of our production e2e tests run in petabox context. That way we can use these UX group test accounts that we can create and protect. Hopefully the BookReader CI can call these (API?) securely and get the benefit without having them in the BookReader repo.

This is pi in the sky...